NotificationApi/NotificationApi.DAL/Services/CreateNotificationService.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NotificationApi.Common;
+using NotificationApi.DAL.Commands;
+using NotificationApi.DAL.Commands.Core;
+using NotificationApi.DAL.Queries;
+using NotificationApi.DAL.Queries.Core;
+using NotificationApi.Domain;
+using Notify.Interfaces;
+
+namespace NotificationApi.DAL.Services
+{
+    public class CreateNotificationService : ICreateNotificationService
```

Comment:
I realised the service was adding to the DAL project. Personally, I don't think the DAL needs to have reference to Notify. This probably belongs in the API project